test/src/main/java/org/apache/accumulo/test/fate/MultipleStoresIT.java

```diff
@@ -249,30 +249,32 @@ private void testMultipleFateInstances(TestStoreFactory<SleepingTestEnv> testSto
     Fate<SleepingTestEnv> fate2 =
         new Fate<>(testEnv2, store2, false, Object::toString, DefaultConfiguration.getInstance());
 
-    for (int i = 0; i < numFateIds; i++) {
-      FateId fateId;
-      // Start half the txns using fate1, and the other half using fate2
-      if (i % 2 == 0) {
-        fateId = fate1.startTransaction();
-        fate1.seedTransaction(TEST_FATE_OP, fateId, new SleepingTestRepo(), true, "test");
-      } else {
-        fateId = fate2.startTransaction();
-        fate2.seedTransaction(TEST_FATE_OP, fateId, new SleepingTestRepo(), true, "test");
+    try {
```

Comment:
I made shutdown() in a finally block for the tests in this class to avoid failures in one test affecting the others. Should have already existed in the test.